src/gtfs/repository/CIFRepository.ts

```diff
@@ -22,8 +22,14 @@ export class CIFRepository {
     private readonly stationCoordinates: StationCoordinates,
     private readonly startRange,
     private readonly endRange,
+    private readonly _excludeFixedLinks: boolean = false,
+    private readonly excludeVstpSchedules: boolean = false,
   ) {}
 
+  get excludeFixedLinks() {
```

Comment:
We don't need this getter. By default, by adding ` private readonly excludeVstpSchedules`, that adds a private field onto the class that is read only, that is also a constructor argument. This is TypeScript syntax shorthand.

We should just take `public readonly excludeFixedLinks: boolean = false` and leave it at that.

Reply:
noted, I have removed the getter in: 2c73484

Reply:
:+1: